### DISCUSSION TITLE
2024-02-16 - Gathering insights from emails - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by synnoveisabel** [SSWConsulting/SSW.Rules.Content#7960] [Chewing the Fat]
Contributers: @GordonBeeming @danielmackay @synnoveisabel 

Hey SSW'ers

Let's talk about EagleEye! 

https://ssw.com.au/rules/gather-insights-from-company-emails/

**Comment by bahjatalaadelssw** [comment]
It would be good for unblocking. I think the checked by stats are only going to be a game and might lose it's critical degree.

**Comment by AlexRothwell** [comment]
There's some great insights already captured and I can imagine more will be on their way. We just need to make sure that there's time allocated to fixing any issues that are made obvious through these reports

**Comment by ozdemir-mehmet** [comment]
Look forward to seeing more metrics being added over time

**Comment by joshbermanssw** [comment]
Stats are awesome... excited to see what other reports are coming

**Comment by ricksu978** [comment]
Suggestion:
Implement "comparison" feature in the report.
e.g.: How was the sales volume Q3 this year "compare" to Q3 last year?

**Comment by cvionnet** [comment]
You're killing it, Isabel!
Emails are a goldmine of information, and EagleEye is the shovel to dig into it.

**Comment by kikibianc** [comment]
Just imagine if the whole company used Zendesk - we could get most of the stats EagleEye has, built in...
For selling for other companies that don't use Zendesk, it's very interesting so far - maybe builtin reports instead of PowerBI? Not all clients use PowerBI, and having reports showing straight away would be "wow".

**Reply by danielmackay** [reply]
We did discuss building more of a SAAS product with built in reports.  However, that is MUCH more expensive to build.  Maybe we'll get in there in the future if the product gains traction.

**Comment by tkapa** [comment]
It'll be interesting to see how this plays out. Like Kiki said, I wonder if there's any software that could have done a lot of this out of the box.

**Reply by danielmackay** [reply]
Where's the fun in that? 😜

**Reply by ozdemir-mehmet** [reply]
SSW EagleEye Pro! ZenDesk Edition

**Comment by TobyChurches** [comment]
Great Video @synnoveisabel !
The stats are a great idea, if we see that it is a small handful of people doing majority of the checks maybe we could try to distribute the responsibility more (where applicable) so that those people don't become blockers.

**Comment by danielmackay** [comment]
Such a great EagleEye video Isabel.  Very clearly presented and explained.  You've done the team proud! 👏

We've only scratched the surface with EagleEye so far.  Emails are one thing, but brining in data on PRs, PBIs, and Deployments (DORA metrics) could be amazing!

Watch this space...

**Comment by BrookJeynes** [comment]
great video from Isabel. Will be interesting to see how this project evolves and how the metrics are used.

**Comment by jimmy1992abc** [comment]
The video and function itself are cool, especially associated with AI.
I don't really like the concept because I think it is a good tool for management, but not for general purpose.

I am not sure about 2 parts:
1. I've used SSWAccounts@ to send many emails, but the report only shows 4.
2. I don't know if the AI will confuse Jim and Jimmy😂, I knew some employees get confused before.

**Comment by Marxoz** [comment]
I like the idea of getting stats out of emails, and throwing AI in there to get some insights. I think there is still a lot more questions we can ask from the emails, but it is a good start

**Comment by jakebayliss** [comment]
Pretty cool to make stats on email. Would be cool to see some trends over a longer period of time and if we can adopt our rules even more

**Comment by isaaclombardssw** [comment]
Definitely interesting to look at the company-wide trends, tonnes of options for future metrics as well.

**Comment by bettybondoc** [comment]
Great presentation, Isabel!

I think it's a great start. Excited to see what other insights we could get using EagleEye.

**Comment by christoment** [comment]
I think it's good to utilise all the available tools to reiterate and improve the system in the company. However, I think it's also important to only use these metrics solely for the sake of analysing and improving the process. Please don't gamify this :(

**Comment by YalalCakirSSW** [comment]
Great video Isabel!

EagleEye is great for getting the stats and visualizing them. With enough data we might predict the future and I would like to see project-based email stats.

**Comment by Harry-Ross** [comment]
Great video by Isabel - will be interesting to see what other reports + features that EagleEye will add in the future. 

+1 to @christoment's comment - I'm hopeful that this won't result in a gamification of the email checking process

**Comment by amankumarrr** [comment]
Great video! Having these statistics really helps motivate people to follow those trends.

**Comment by zacharykeeping** [comment]
Great video! The current reports look useful and will be cool to see what else comes from this.

**Comment by Dhruv-0987** [comment]
Awesome video Isabel short and simple, I think EagleEye is great with useful insights, maybe we can have a CTF explaining how it actually works and how we can use that in other projects.

**Reply by danielmackay** [reply]
@Dhruv-0987 - if Adam's talk from NDC get's published to YouTube, he has a section where he covers how it works.

**Comment by SamBWagner** [comment]
Excellent video and the team working on the project seems nice and lean. I'm still unconvinced of the project at large but I'm excited to see a fully fledged MVP built

**Comment by AshAnil** [comment]
The stats are cool. Looking forward to see more stats.

**Comment by JeanThirion** [comment]
I'm still a bit creeped out by this... Named data is invading privacy in a way, but the data is very good to have. So yeah, I'm leaning towards liking it a lot, it's just a change of mentality.

**Comment by BenDunkerleySSW** [comment]
Very keen to see how the data can be applied & help us learn how to solve issues.

**Comment by chrisschultzssw** [comment]
There's a lot of potential here to get data and insights from other sources - looking forward to seeing what comes next.

**Comment by jaydenalchin** [comment]
As a UX expert i'm all for gathering insights to make evidence-based decisions. Having said that, I find it frustrating that time has been spent on developing this when we already know where the bottlenecks are. Assuming we didn't, there are easier ways we could have gathered this data as others have suggested.

**Reply by JackDuan123** [reply]
As a Developer who is also reading and learning a lot in Philosophy and Anthropology, to get some insights from data and make evidence-based decisions is not actually edidence-based, we are all living in reality and based on the same reality we have different opinions and feelings, from those opinions and feelings we make different decisions.

**Comment by stephanfakossw** [comment]
Pretty cool reports! The percentage columns seem to be broken in all of them - hopefully not hard to fix...

**Comment by JackDevAU** [comment]
Cool reports - I'm with @jaydenalchin on this though. We don't need reports that find issues when we already know where they are and what's causing them. Stuff like the checked by are interesting though!

**Comment by JackDuan123** [comment]
Comment on Jayden's comment:
```As a UX expert i'm all for gathering insights to make evidence-based decisions. Having said that, I find it frustrating that time has been spent on developing this when we already know where the bottlenecks are. Assuming we didn't, there are easier ways we could have gathered this data as others have suggested```:


As a Developer who is also reading and learning a lot in Philosophy and Anthropology, to get some insights from data and make evidence-based decisions is not actually edidence-based, we are all living in reality and based on the same reality we have different opinions and feelings, from those opinions and feelings we make different decisions.

I reckon eagle eye is a fun project but I would want to see something more fancy ranther than just a barchart. Like a connect graph to see more, email checked is too simple, and it do not have enough insights something can be improved:

1. who is checking the most emails to client
2. who is checking decision making emails
3. who check who's email the most frequent?
4. who covers the most people on checking email?
5. How many people checked rules via email?
6. Generating a connection graph from emails and see how people interact with each other

**Comment by leahy268** [comment]
Great video.  Interesting look at using email data for stats.  Perhaps it could be extended to Teams?  
Perhaps it could also look at work hours and how many people are working outside of work time..  I know Viva already does some of that but a custom version could be interesting.

**Comment by MattParkerDev** [comment]
Enforcement via metrics and and big brother seems like a poor alternative to fostering a culture where getting emails checked is understood and valued.

**Reply by JackDuan123** [reply]
Good one!

Mordern Captalist is going to a SU, we have no way to escape, lol

**Comment by drwharris** [comment]
A very wise person once told me that metrics are only as good as the person interpreting them. He then went on to say that they are only as useful as the person empowered to act upon them.

**Comment by Freego1783** [comment]
Great video Isabel!

Stats are cool, but privacy is nice too. We're not there yet but there's probably a line we don't wanna cross. 

I'm curious to see what other reports we could get with EagleEye though!

**Comment by ozair-ashfaque** [comment]
Awesome video Isabel !!
Great beginning. I am curious to see what other reports we can get out of EagleEye

**Comment by vladislav-kir** [comment]
Great video from Isabel, and I quite like those reports :)

**Comment by Ben0189** [comment]
Data is one of the most important things for a company (especially Large Enterprises). Eagle Eye is a great tools and collecting insight from email is a great starting point. 

In my opinion, if we can make Eagle eye not tailored specifically to SSW needs but can easily adapt to different companies, it will be a valuable SAAS product. It can be a tools that can gather insight from various company daily uses software (such as Teams, emails, timesheet system.....)

**Comment by sethdaily** [comment]
Great stats, looking forward to seeing more added

**Comment by jeoffreyfischer** [comment]
I think it's a great product that has a lot of potential!

**Comment by 2fernandez** [comment]
Awesome video, @synnoveisabel; you are such a great presenter.

I love this idea of monitoring emails.
I wish we could Gamify some of these results by adding Ranks😉  it might motivate users, for example, to improve their response times.

@matt-goldman what do you think?

![Screenshot 2024-02-19 at 1 01 47 pm](https://github.com/SSWConsulting/SSW.Rules.Content/assets/27670973/61560a4d-464c-4c41-b35b-ae5bc1d34197)
**Figure: Dark mode Email stats**

**Reply by matt-goldman** [reply]
Good idea. I like the idea of gamification if it motivates people. It can be negative when it shows you unachievable goals but this is definitely an example where people can see an opportunity to improve.

**Comment by AttackOnMorty** [comment]
It's nice to have the report built upon the rule we intend to follow, e.g. "Asked for Checked By" report.

We should focus more on the reports we can gain real insights and action upon.

**Comment by MarcusIrmscherSSW** [comment]
Great video Isabel. Very well presented.  👍👍

**Comment by tino-liu** [comment]
Great video! The information provided is clear. I can't wait to see more features being implemented in the future.

**Comment by Aibono1225** [comment]
Great video and tool 👍. Looking forward to seeing what EagleEye can be used to generate other useful stats.

**Comment by PothieuG** [comment]
Excellent presentation by Isabelle!
I have always loved stats and charts, and I am excited to see new tools emerging in this area. I like the idea of having general statistics; I am less fond of statistics targeted at individuals.

**Comment by AntPolkanov** [comment]
It was interesting to look at the stats!
Apart from Total, can you please add the 5-number summary: max, min, median, quartile 1 and quartile 3. That would simplify analysis of the distribution.

**Comment by matt-goldman** [comment]
Top tier video from Isobel!

Regarding EagleEye itself, I'm concerned we're "done a Microsoft" here (think Tablet PC, HoloLens, etc) where we've shown off the _potential_ of a technology too early, allowing others to come along and iterate on the concept. I think if we waited until we had _actual_ tangible improved results arising from this, that would be a true mic-drop moment and would make people pay attention.

**Comment by tombui99** [comment]
EagleEye would be great especially for unblocking. Being able to see who's free or not is a must.

**Comment by jernejk** [comment]
I love the idea that we are investing in unblocking people, and to do that, we'll have to track meetings, calls, and at least the number of messages sent to people.

I often get distracted because people need my help, usually coding or better understanding a product or development principles. That usually impacts the deliverables to the client, such as code and better PBIs. I see many people doing the same.

We should track Azure DevOps and GitHub, especially the code quality, how much people pair programs or help debugging, review and comment PR, refine backlogs, manage backlogs, report bugs, etc. All the work that is extremely important daily is to deliver value to clients' users.

It would be great also to acknowledge people who do the work that keeps the application stable; teams work like a well-oiled machine and upskill everyone to improve.

**Comment by jimmidier** [comment]
Great video and nice work! Love to see more stats coming up in the near future

**Comment by PennyWalker** [comment]
Very cool. I like the fact that we are collecting data that Managers can use to help spread the workload. However, it will only be useful if we use this data to make changes and then monitor how those trends are changing over time to make sure it's effective.

Also, I can't believe that I beat Adam in the sent emails category! 👑😂

**Comment by awaltos** [comment]
Having insights are a great idea. Using AI to filter through them and provide an overview can be very powerful. The key will be to find valuable insights and be able to track and compare them, where those insights lead to meaningful change, i.e. reduce time wastage, identify problem areas, etc.

**Comment by babakamyljanovssw** [comment]
Great Video! Nice to have Data Analysis project that could be helpful to predict the future and excited for the new upcoming reports from EagleEye.

**Comment by jackreimers** [comment]
Very insightful video, I can see some great insights coming out of EagleEye!

**Comment by rubycogan** [comment]
I am loving seeing AI incorporated into more and more aspects of work at SSW!
